packages/graphql/src/translate/create-projection-and-params.ts

```diff
@@ -251,12 +251,14 @@ function createProjectionAndParams({
         }
 
         if (relationField) {
+            // WARN: In some test this node is undefined. If this is intended, we should update typings to reflect that
```

Comment:
Can you explain what you mean by this?

Reply:
Our typings are casting the variable `referenceNode` as type Node. But in some tests (22) this variable is `undefined`.

Sadly, I'm not sure (due to lack of context) if this is an error on our tests (e.g. A mock not being properly set) or if it is intended and thus, we should update the type to be `Node | undefined` (which, in turn, requires further updates on the code.

I'll update the comment, to try and explain myself better. Or do you think I should remove the comment altogether?

Reply:
Honestly, I think that's just laziness in writing tests more than anything - there's no problem with the production code itself as we wouldn't have gotten to the translation step if `referenceNode` was `undefined`.

I think definitely remove the comment, perhaps just keep a note of this in Trello or something instead.